combat-trainer.lic

```diff
@@ -2009,7 +2009,8 @@ class TrainerProcess
 
   def pickbox(game_state)
     return unless @pet_box_source && @pet_count > 0
-    return if game_state.loaded || game_state.weapon_is_summoned? || game_state.offhand?
+    return if game_state.loaded || game_state.offhand?
+    return unless game_state.weapon_is_summoned?
```

Comment:
❓ Does this mean 'return if weapon is not summoned'? That seems backwards. Or is the logic for `weapon_is_summoned?` inverted?

Reply:
Yeah, this is weird. I wanted to use the `summoned_info` method so I had it all weird. Put a !nil? return for the method, so it can move back up to `if summoned`.